MODULE.bazel

```diff
@@ -51,6 +52,16 @@ use_repo(bazel_lib_toolchains, "yq_linux_arm64")
 use_repo(bazel_lib_toolchains, "yq_linux_ppc64le")
 use_repo(bazel_lib_toolchains, "yq_linux_s390x")
 use_repo(bazel_lib_toolchains, "yq_windows_amd64")
+use_repo(bazel_lib_toolchains, "bsd_tar_toolchains")
+
+register_toolchains(
```

Comment:
do we need this here? toolchains are already registered by bazel-lib

Reply:
Looks like it is required. Without it I get,
```
ERROR: /Users/greg/aspect/rules/js/rules_js/BUILD.bazel:70:17: While resolving toolchains for target //:.aspect_rules_js/node_modules/@mycorp/pkg-b (3ac6104): No matching toolchains found for types @@aspect_bazel_lib~1.41.0//lib:tar_toolchain_type.
To debug, rerun with --toolchain_resolution_debug='@@aspect_bazel_lib~1.41.0//lib:tar_toolchain_type'
If platforms or toolchains are a new concept for you, we'd encourage reading https://bazel.build/concepts/platforms-intro.
```

Reply:
Ohhh. but only on bazel-lib 1.x and not 2.x.. that implies that I'm missing something in bazel-lib 1.x for toolchain registration.

Reply:
Fixed with bazel-lib v1.42.0